patterns/hidden-post-meta.php

```diff
@@ -6,22 +6,24 @@
  */
 ?>
 
-<!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0.3em"}},"layout":{"type":"flex","justifyContent":"left"}} -->
-<div class="wp-block-group"><!-- wp:post-date {"format":"M j, Y","isLink":true} /-->
+<!-- wp:group {"style":{"layout":{"flexSize":null,"selfStretch":"fit"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+	<!-- wp:group {"style":{"spacing":{"blockGap":"0.3em"}},"layout":{"type":"flex","justifyContent":"left"}} -->
+	<div class="wp-block-group">
+		<!-- wp:post-date {"format":"M j, Y","isLink":true} /-->
 
-<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast-2"}}}},"textColor":"contrast-2"} -->
-<p class="has-contrast-2-color has-text-color has-link-color">—</p>
-<!-- /wp:paragraph -->
+		<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast-2"}}}},"textColor":"contrast-2"} -->
+		<p class="has-contrast-2-color has-text-color has-link-color">—</p>
```

Comment:
I don't understand why this would need a link color.

Reply:
Are you referring to the dash `—`? If yes, you can remove the link color.

Reply:
This is being added by the editor for some reason everytime you add a color to a paragraph and I don't know if it's a bug

Reply:
Also, I changed the way the post meta was built because it was breaking horribly on smaller viewports, keep that in mind